requirements/test.in

```diff
@@ -0,0 +1,3 @@
+coverage
+edx-opaque-keys
+mock
```

Comment:
Maybe just use `unittest.mock` instead now that we're on Python 3?

Reply:
Done.